en/docs/get-started/how-to-guides/ai-data-mapping.md

```diff
@@ -0,0 +1,487 @@
+        */
+        export function mapFunction(input: Root): OutputRoot {
+            return {
+                weather: {
+                    main: input.weather[0].main,
+                    description: input.weather[0].description.toLocaleUpperCase(),
+                    icon: input.weather[0].icon,
+                    daylight_hours: (input.sys.sunset - input.sys.sunrise) / 3600,
+                },
+                temp: {
+                    temp: input.main.temp - 273.15,
+                    temp_description: input.main.temp > 30 ? "Hot" : input.main.temp < 15 ? "Cold" : "Moderate",
+                },
+                wind: {
+                    speed: input.wind.speed,
+                    deg: input.wind.deg,
+                },
+                pressure: input.main.pressure,
+                humidity: `${input.main.humidity}%`,
+                visibility: input.visibility,
+            };
+        }
+        ```
+
+### Step 4: Build and Run
+
+1. Click on the **Build and Run** button to build and deploy the integration.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-build.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-build.png" alt="build and run" width="80%"></a>
+
+2. **Runtime Services** will open after the server is up and running and from that panel select the API that you need to try out using the **Try It** button.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-tryit.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-tryit.png" alt="try it out" width="80%"></a>
+
+3. Click on the **Try it Out** button to provide the required query parameters.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-operation-tryit.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-operation-tryit.png" alt="try it out operation" width="80%"></a>
+
+4. Provide the required parameters and click on the **Execute** button to invoke the API.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-request.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-request.png" alt="execute request" width="80%"></a>
+
+5. Check the response received from the server and confirm that the weather details for the specified city have been mapped to the output data schema you provided.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-response.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-response.png" alt="response" width="80%"></a>
+
+Data mapping in seconds. Try out this and our other cool AI features, and see for yourself how much easier integration development can be.
```

Comment:
how about `Try this feature along with our other exciting AI capabilities and discover how much simpler integration development can be.`

Reply:
Fixed with 26b89fe